app/mailers/emailer.rb

```diff
@@ -124,7 +126,7 @@ def draft_application_saved(params)
 
   def format_app_due_date(listing)
     due = Time.zone.parse(listing['Application_Due_Date'])
-    due_time = "#{due.strftime('%l')}:#{due.strftime('%M')} #{due.strftime('%p')}"
+    due_time = "#{due.strftime('%-l')}:#{due.strftime('%M')} #{due.strftime('%p')}"
```

Comment:
documentation: https://www.tigraine.at/2015/07/28/ruby-timestrftime-pads-e-with-whitespace

looks like one of the package updates included whitespace padding in front of our date, messing with our tests

Reply:
thanks goddess for tests